iamy/cfn_test.go

```diff
@@ -30,6 +30,9 @@ func TestCfnMangedResources(t *testing.T) {
 		if !cfn.IsManagedResource(CfnIamPolicy, "foobar-ABCDEFGH1234567") {
 			t.Fatal("names with id suffix are managed")
 		}
+		if !cfn.IsManagedResource(CfnIamPolicy, "foobar-ABCDEFabcde12345") {
```

Comment:
Can we add a test in here to show that it doesn't catch something like foobar-Production ?